io.github.giantpinkrobots.varia.json

```diff
@@ -0,0 +1,155 @@
+{
+    "app-id" : "io.github.giantpinkrobots.varia",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "45",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "varia",
+    "_justifications": "This program utilizes aria2c, and aria2c requires the networking permission and the xdg-download directory in order to download files.",
```

Comment:
this property is invalid. Use `//`

Reply:
Using // made the json file invalid, and I could no longer build off of it locally.

Reply:
```suggestion
    "//": "This program utilizes aria2c, and aria2c requires the networking permission and the xdg-download directory in order to download files.",
```

like that

Reply:
but then a README is usually more appropriate